index.html

```diff
@@ -21,14 +21,8 @@
       new Chartist.Line('.ct-chart', {
         labels: ['1', '2', '3', '4', '5', '6'],
         series: [
-          {
-            name: 'Fibonacci sequence',
-            data: [1, 2, 3, 5, 8, 15]
-          },
-          {
-            name: 'Golden section',
-            data: [1, 1.618, 2.618, 4.236, 6.854, 11.09]
-          }
+          [1, 2, 3, 5, 8, 15],
+          [1, 1.618, 2.618, 4.236, 6.854, 11.09]
```

Comment:
@karlhorky the structure you used make sense, but as much as I checked the ["offical" examples](https://gionkunz.github.io/chartist-js/examples.html) I couldn't find it in this format.